### DISCUSSION TITLE
dont receive tokens

### DISCUSSION THREAD

**Discussion by azakaragulov** [Chia-Network/chia-blockchain#18404] [Support]
Hello,

I have sent some tokens from the OKX exchange and so far, they have not been credited to my balance.

Could you please help me to check if there is any issue with the transaction?

Thank you for your assistance.

My funds were transferred from OKX using the blockchain, but they have not arrived in my account on this platform. The details of the transfer are as follows:
Криптовалюта: XCH
Сумма к получению: 13,364366 XCH
Комиссия: 0,0004 XCH
Сеть: Chia
Платформа вывода средств: OKX
Адрес: xch15795j0tw7ynnzj6c9k8hj24d26jhzv3sy46h9lptsn8tr4yhdx9sy28nlk
TXID: 0x58faa0546bc34aee111b4899bac4068f3746e123bd478dbb4bd2c22e9e3acb5f

![photo_2024-08-05_23-22-27](https://github.com/user-attachments/assets/dc803769-e030-4a60-af50-f60b09764960)

**Comment by BrandtH22** [comment]
Hey @azakaragulov , , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

What I recommend trying is a resync of the wallet:

 - If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync.
 - If using the cli then I would stop chia, rename the wallet db folder in `~.chia\mainnet\wallet\db` then restart chia.

If the funds are still not visible can you let us know:
- if the wallet icon in the top right is orange or green?
- if orange can you click on the wallet icon and let  us know if you see any connected peers?
- can you provide your log file from `~\.chia\mainnet\log\debug.log`

**Comment by azakaragulov** [comment]
Thanks, the coins have arrived and are displayed.
